src/screens/earn/farm-item/farm-item.tsx

```diff
@@ -106,13 +117,25 @@ export const FarmItem: FC<Props> = ({ farm, lastStakeRecord }) => {
         kind: OpKind.TRANSACTION
       }));
 
-      dispatch(
-        navigateAction(ModalsEnum.Confirmation, {
-          type: ConfirmationTypeEnum.InternalOperations,
-          opParams,
-          testID: 'CLAIM_REWARDS'
-        })
-      );
+      if ((lastStakeRecord?.rewardsDueDate ?? 0) > Date.now()) {
+        Alert.alert(
```

Comment:
I doubt whether we need this modal at all. The questions is to requirements but not to you: https://madfishgroup.slack.com/archives/C0535E9N3PE/p1684509695466739